services/movie.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+from typing import Optional
+
+
+def get_movies(
+        genres_ids: list[int] | None = None,
+        actors_ids: list[int] | None = None,
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
```

Comment:
you don't need it because if both params are passed, then both if statement below will be activated